### DISCUSSION TITLE
Reworked my init files.  Now hitting tab gives error

### DISCUSSION THREAD

**Discussion by pedz** [emacs-helm/helm#2565] [Q&A] [answered]
I'm hoping you can point out where I goofed.

I reworked all of my emacs start up files but the one thing that hasn't worked since then is tab.  While I still hit tab when I should not, it use to give me a list of actions.  Now it just beeps and gives me.

> helm-fuzzy-default-highlight-match-1: Wrong type argument: stringp, helm-find-file-or-marked

For example, I do a `C-x C-f (helm-find-files)` and type a part of the path and hit tab (when I should hit `C-j`) and it use to give me a list of actions but now it beeps with an error.

Any ideas of what I messed up?

tab is bound to `helm-select-action`

**Comment by pedz** [comment]
I checked out commit 587be477 (Date:   Thu Sep 15 11:24:52 2022 +0200) and it works now.  This was the commit that I was using before.

So, some change between commit 587be477  commit 2e33e343 (Date:   Sun Oct 23 08:06:17 2022 +0200) is causing the problem.

I don't know the emacs debugger well enough to help much.  I set `debug-on-error` and trap into the debugger but somehow helm has the keyboard all fouled up and the debugger commands don't work.

This should possibly be made into a bug report instead of a discussion.

**Comment by pedz** [comment]
Ok.  I checked out tag v3.8.8 and it is working.  I finally figured out that I'm using code that isn't released yet.  Sorry for the noise.

**Comment by thierryvolpiatto** [comment, marked as answer]
Perry Smith ***@***.***> writes:

> I'm hoping you can point out where I goofed.
>
> I reworked all of my emacs start up files but the one thing that hasn't worked since then is tab. While I still hit tab when I should not, it use to give me a list of actions. Now it just
> beeps and gives me
>
>     helm-fuzzy-default-highlight-match-1: Wrong type argument: stringp, helm-find-file-or-marked
>    
> Any ideas of what I messed up?

You messed nothing, it was a bug related to my recent changes, it should
be fixed now.

-- 
Thierry
